docs/getting_started.md

```diff
@@ -1,51 +1,89 @@
 # Getting Started
 
-#### Trin is currently in unstable alpha, and should not be used in production. If you run into any bugs while using Trin, please file an Issue!
-#### Check out the [Release Notes](/docs/release_notes.md) to see the latest supported features.
+**Trin is currently in unstable alpha, and should not be used in production. If you run into any bugs while using Trin, please file an Issue!**
 
-# Pre-requisites
-- You will need an [Infura](https://infura.io/) ID to use Trin.
-- You will need [Rust](https://www.rust-lang.org/) installed on your machine.
+**Check out the [Release Notes](/docs/release_notes.md) to see the latest supported features.**
 
-# Via Source
+## Prerequisites
+- [Infura](https://infura.io/) project ID
+- [Rust](https://www.rust-lang.org/) installation
 
-### Install dependencies (on Ubuntu/Debian)
+## Building, Testing, and Running
+
+Note: If you use a VPN, you should disable it before running Trin.
+
+Install dependencies (Ubuntu/Debian):
 
 ```sh
 apt install libssl-dev librocksdb-dev libclang-dev 
 ```
 
+Environment variables:
+
+```sh
+# Required 
+export TRIN_INFURA_PROJECT_ID=<infura-project-id>
+
+# Optional 
+export RUST_LOG=<error/warn/info/debug/trace>
+export TRIN_DATA_PATH=<path-to-data-directory>
+```
+
+Build, test, and run:
+
 ```sh
 git clone https://github.com/ethereum/trin.git
 cd trin
-export TRIN_INFURA_PROJECT_ID=yourinfurakey
-RUST_LOG=debug cargo run -p trin
+
+# Build
+cargo build --workspace
+
+# Run test suite
+cargo test --workspace
+
+# Build and run test suite for an individual crate
+cargo build -p trin-core
+cargo test -p trin-core
+
+# Run
+cargo run -p trin
 ```
 
-(Trin might take a couple of minutes to compile the first time you launch it.)
+Note: You may also pass environment variable values in the same command as the run command. This is especially useful for setting log levels.
```

Comment:
Is it worth listing the available log levels? And explain how `debug` can be quite overwhelming?

Reply:
I tried to address the first note above in the environment variables section. I'm not sure that it's worth making a note about `debug`, because I think that generally holds true for most software.